avocado/utils/gdb.py

```diff
@@ -59,11 +59,11 @@
 REMOTE_TRANSMISSION_FAILURE = '-'
 
 #: How the remote protocol flags the start of a packet
-REMOTE_PREFIX = '$'
+REMOTE_PREFIX = b'$'
```

Comment:
This commit shouldn't be merged with the previous one?

Reply:
One reason would mean because this change is used in the previous commit, but I did not find that to be the case.  Or do you mean because of a commit/granularity style?